src/NzbDrone.Core/Download/Clients/uTorrent/UTorrent.cs

```diff
@@ -165,7 +166,7 @@ public override IEnumerable<DownloadClientItem> GetItems()
                 }
 
                 // 'Started' without 'Queued' is when the torrent is 'forced seeding'
-                item.IsReadOnly = torrent.Status.HasFlag(UTorrentTorrentStatus.Queued) || torrent.Status.HasFlag(UTorrentTorrentStatus.Started);
+                item.CanBeRemovedFromClient = !(torrent.Status.HasFlag(UTorrentTorrentStatus.Queued) || torrent.Status.HasFlag(UTorrentTorrentStatus.Started));
```

Comment:
I'd use: `!torrent.Status.HasFlag(UTorrentTorrentStatus.Queued) && !torrent.Status.HasFlag(UTorrentTorrentStatus.Started)` but that's just preference.